lib/web/apiserver_test.go

```diff
@@ -2481,6 +2483,126 @@ func TestCheckAccessToRegisteredResource(t *testing.T) {
 	}
 }
 
+func TestAuthExport(t *testing.T) {
+	ctx := context.Background()
+
+	env := newWebPack(t, 1)
+	clusterName := env.server.ClusterName()
+
+	proxy := env.proxies[0]
+	pack := proxy.authPack(t, "test-user@example.com", nil)
+
+	validateTLSCertificateDERFunc := func(t *testing.T, b []byte) {
+		cert, err := x509.ParseCertificate(b)
+		require.NoError(t, err)
+		require.NotNil(t, cert)
+		require.Equal(t, cert.Subject.CommonName, "localhost", "unexpected certificate subject CN")
+	}
+
+	validateTLSCertificatePEMFunc := func(t *testing.T, b []byte) {
+		pemBlock, _ := pem.Decode(b)
+		require.NotNil(t, pemBlock)
+
+		validateTLSCertificateDERFunc(t, pemBlock.Bytes)
+	}
+
+	for _, tt := range []struct {
+		name           string
+		authType       string
+		expectedStatus int
+		assertBody     func(t *testing.T, bs []byte)
```

Comment:
Since we are now testing under ca_export_test.go, and we do have strong body assertions there (checking DER/PEM/ssh keys), I'd be alright with going lighter with body assertions here. Your call.

Reply:
I think I'm going to keep them